website/content/en/docs/Community/Governance/_index.md

```diff
@@ -0,0 +1,90 @@
+  but they do not have any access to merge pull requests nor push commits.
+  A Reviewer is considered as a candidate of Committers.
+  Reviewers are not recognized as Maintainers in <https://github.com/cncf/foundation/blob/main/project-maintainers.csv>.
+
+See also the [Contributing](../contributing) page.
+
+### Current maintainers
+
+| Name               | Role      | GitHub ID (not X ID)                           | GPG fingerprint                                                                          |
+|--------------------|-----------|------------------------------------------------|------------------------------------------------------------------------------------------|
+| Akihiro Suda       | Committer | [@AkihiroSuda](https://github.com/AkihiroSuda) | [C020 EA87 6CE4 E06C 7AB9  5AEF 4952 4C6F 9F63 8F1A](https://github.com/AkihiroSuda.gpg) |
+| Jan Dubois         | Committer | [@jandubois](https://github.com/jandubois)     |                                                                                          |
+| Anders F Björklund | Committer | [@afbjorklund](https://github.com/afbjorklund) |                                                                                          |
+| Balaji Vijayakumar | Committer | [@balajiv113](https://github.com/balajiv113)   |                                                                                          |
+
+<!-- TODO: invite non-committer reviewers -->
+
+### Addition and promotion of Maintainers
+An active contributor to the project can be invited as a Reviewer,
+and can be eventually promoted to a Committer after 2 months at least.
+
+A contributor who have made significant contributions in quality and in quantity
+can be also directly invited as a Committer.
+
+A proposal to add or promote a Maintainer must be approved by 2/3 of the Committers who vote within 7 days.
+
+A proposal should happen as a GitHub pull request to the Maintainer list above.
+It is highly suggested to reach out to the Committers before submitting a pull request to check the will of the Committers.
+
+### Removal and demotion of Maintainers
+A Maintainer who do not show significant activities for 6 months, or, who have been violating the Code of Conduct,
+may be demoted or removed from the project.
+
+A proposal to demote or remove a Maintainer must be approved by 2/3 of the Committers (excluding the person in question) who vote within 14 days.
+
+A proposal may happen as a GitHub pull request, or, as a private discussion in the case of removal of a harmful Maintainer.
+It is highly suggested to reach out to the Committers before submitting a pull request to check the will of the Committers.
+
+### Other decisions
+Any decision that is not documented here can be made by the Committers.
+When a dispute happens across the Committers, it will be resolved through a majority vote within the Committers.
```

Comment:
Is there a tie-breaker, or is any tie a failed vote?

Reply:
I guess it should be considered a failed vote?
Basically we should try to reach consensus

Reply:
Added "A tie should be considered as a failed vote."